tests/test_logger.py

```diff
@@ -90,3 +91,15 @@ def get_mock_tqdm(position: int, *args, **kwargs):
     for mock_tqdm in is_train_to_mock_tqdms[False]:
         assert mock_tqdm.update.call_count == trainer._eval_subset_num_batches
         mock_tqdm.close.assert_called_once()
+
+
+@pytest.mark.parametrize("world_size", [
+    pytest.param(1),
+    pytest.param(2, marks=pytest.mark.world_size(2)),
+])
+@pytest.mark.timeout(10)
+def test_wandb_logger(mosaic_trainer_hparams: TrainerHparams, world_size: int):
```

Comment:
Is this just to verify that the logger doesn't crash?

Reply:
Yea, we didn't have a wandb logger test before. This just verifies that it's initialized properly, etc...